examples/riot/pkg_libcoap/Makefile

```diff
@@ -0,0 +1,15 @@
+PKG_NAME=libcoap
+PKG_URL=https://github.com/mrdeep1/libcoap
```

Comment:
This might be a bit misleading if people retrieved this code from `obgm/libcoap`, and now the RIOT version they use lives at `mrdeep1/libcoap`. "People" also includes developers who wonder why CI might report different things for their code than expected.

Reply:
Unfortunately it has to be this until merged into obgm/libcoap (as the hash containing RIOT fixes only exists under mrdeep1), at which point there will be a quick PR to point `PKG_URL` and `PKG_VERSION` to the right places.